src/components/Items/ProfileContact.js

```diff
@@ -14,46 +18,75 @@ const ProfileContact = ({
   showArrow,
   userId,
   item,
-  ImageComponent = null
-}) => (
-  <Pressable
-    onPress={onPress}
-    android_ripple={{
-      color: COLORS.gray1,
-      borderless: false,
-      borderRadius: 10
-    }}
-    style={styles.pressable}>
-    <View style={styles.container}>
-      <View style={styles.profile}>
-        {ImageComponent || (
-          <Image
-            testID="image"
-            style={styles.image}
-            source={{uri: photo !== '' ? photo : undefined}}
-          />
-        )}
-        <Text style={styles.fullname}>{fullname}</Text>
-      </View>
-      {showArrow && (
-        <>
-          {userId !== item.user_id && (
-            <View>
-              <Mi name="arrow-forward-ios" size={18} />
-            </View>
+  ImageComponent = null,
+  disabled = false,
+  from
+}) => {
+  const handleYouText = () => {
+    if (!from && (!isContainUrl(item?.user?.image) || item?.user?.name === ANONYMOUS_USER)) {
+      return '(You)';
+    }
+    return '';
+  };
+
+  return (
+    <Pressable
+      onPress={onPress}
+      android_ripple={{
+        color: COLORS.gray1,
+        borderless: false,
+        borderRadius: 10
+      }}
+      disabled={!showArrow || disabled}
+      style={styles.pressable}>
+      <View style={styles.container}>
+        <View style={styles.profile}>
+          {ImageComponent || (
+            <Image
+              testID="image"
+              style={styles.image}
+              source={{uri: photo !== '' ? photo : undefined}}
+            />
           )}
-        </>
-      )}
-      {select && (
-        <View testID="selected">
-          <MemoIc_Checklist />
+          <Text testID="name" style={styles.fullname(userId === item?.user_id)}>
+            {fullname} {handleYouText()}
+          </Text>
         </View>
-      )}
-    </View>
-  </Pressable>
-);
+        {showArrow && (
+          <>
+            {userId !== item.user_id && (
+              <View>
+                <Mi name="arrow-forward-ios" size={18} />
+              </View>
+            )}
+          </>
+        )}
+        {select && (
+          <View testID="selected">
+            <MemoIc_Checklist />
+          </View>
+        )}
+      </View>
+    </Pressable>
+  );
+};
+
+ProfileContact.propTypes = {
+  disabled: PropTypes.bool,
+  item: {
+    user_id: PropTypes.string
+  },
+  photo: PropTypes.string,
+  fullname: PropTypes.string,
+  onPress: PropTypes.func,
+  select: PropTypes.bool,
+  showArrow: PropTypes.bool,
+  userId: PropTypes.string,
+  ImageComponent: PropTypes.node,
+  from: PropTypes.string
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

PropTypes have been added for the component props. This is a good practice as it helps validate the props data type and shows a warning if the prop does not match the expected data type. However, the PropTypes for `item` seems to be incorrect. It should be `PropTypes.shape` instead of just an object.

```diff
-  item: {
-    user_id: PropTypes.string
-  },
+  item: PropTypes.shape({
+    user_id: PropTypes.string
+  }),
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->